frb_dart/lib/src/basic.dart

```diff
@@ -164,6 +177,19 @@ abstract class FlutterRustBridgeWireBase {
 
 class _CloseStreamException {}
 
+class Backtrace {
+  Backtrace(this.backtrace);
+  final String backtrace;
+  @override
+  String toString() => backtrace;
+}
+
+class FrbException implements Exception {}
+
+abstract class FrbBacktracedException {
+  String? get backtrace;
```

Comment:
nit: shall we make it `String?` or `String`

Reply:
For cases when the error has no backtrace field, it should be null

Reply:
I guess for no backtrace field, it should *not* `implements` FrbBacktracedException, and only implements FrbException?